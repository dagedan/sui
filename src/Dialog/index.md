
## Dialog

Demo:

```tsx
import React from 'react';
import { Dialog } from 'sui';

export default () => <Dialog>sdf老师款到发货考虑好</Dialog>;
```

More skills for writing demo: https://d.umijs.org/guide/basic#write-component-demo
