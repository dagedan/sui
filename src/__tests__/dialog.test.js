import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Dialog from '../Dialog';

Enzyme.configure({ adapter: new Adapter() });

describe('Dialog组件相关', () => {
  it('Dialog渲染样式正常', () => {
    const wrapper = shallow(<Dialog />);
    // expect(wrapper).toMatchSnapshot();// 进行快照保存
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
