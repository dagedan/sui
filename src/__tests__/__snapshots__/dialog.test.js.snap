// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dialog组件相关 Dialog渲染样式正常 1`] = `"<div className=\\"biz_dev_dialog\\" onClick={[undefined]} />"`;
